ArcGISRuntimeSDKQt_QMLSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.qml

```diff
@@ -316,10 +316,10 @@ Rectangle {
                             text: "Hide directions"
                             onClicked: {
                                 if (text === "Hide directions") {
-                                    directionsView.delegate = blankDelegate;
+                                    directionsView.visible = false;
```

Comment:
Optional - you could just bind the ListView's visibility to this buttons text string